tools/container/toolkit/toolkit.go

```diff
@@ -190,6 +194,13 @@ func Flags(opts *Options) []cli.Flag {
 			Destination: &opts.cdiKind,
 			EnvVars:     []string{"CDI_KIND"},
 		},
+		&cli.StringFlag{
+			Name:        "toolkit-config-source",
+			Usage:       "The file where the NVIDIA Container toolkit source configuration is specified",
+			Value:       nvidiaContainerToolkitConfigSource,
+			Destination: &opts.toolkitConfigSource,
+			EnvVars:     []string{"TOOLKIT_CONFIG_SOURCE"},
+		},
```

Comment:
Should this really be required at present?

Reply:
I can remove this